Build failure in Julia v0.6

```julia
gcc: error: $: No such file or directory
make: *** [Makefile:12: task] Error 1
===============================[ ERROR: Turing ]================================

LoadError: failed process: Process(`make`, ProcessExited(2)) [2]
while loading /home/juliohm/.julia/v0.6/Turing/deps/build.jl, in expression starting on line 1

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: Turing had build errors.

 - packages with build errors remain installed in /home/juliohm/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("Turing")`
 - build a single package by running its `deps/build.jl` script

================================================================================
```

--- a/deps/Makefile
+++ b/deps/Makefile
@@ -5,8 +5,8 @@ CXXFLAGS += $(shell julia $(JL_SHARE)/julia-config.jl --cflags)
 LDFLAGS  += $(shell julia $(JL_SHARE)/julia-config.jl --ldflags)
 LDLIBS   += $(shell julia $(JL_SHARE)/julia-config.jl --ldlibs)
 
-CFLAGS   += $(shell julia -e 'print("-std=gnu99 ")')$
-CFLAGS   += $(shell julia -e 'if is_windows() && Int == Int32 print("-march=pentium4 ") end')$
+CFLAGS   += $(shell julia -e 'print("-std=gnu99 ")')
+CFLAGS   += $(shell julia -e 'if is_windows() && Int == Int32 print("-march=pentium4 ") end')
 
 task: task.c
 	$(CC) $(CFLAGS) -O2 -shared -fPIC task.c $(LDFLAGS) $(LDLIBS) -o libtask.$(LIBEXT)

